scripts/utils/sign_and_send.js

```diff
@@ -73,7 +67,44 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     console.log("Transaction awaiting execution in the interface", interfaceLink)
   }
 
+  /**
+   * Checks that a transaction was already proposed to the gnosis-safe UI
+   *
+   * @param {Address} masterSafe Address of the master safe owning the brackets
+   * @param {Transaction} transaction The transaction to be signed and sent
+   * @param {string} network either rinkeby or mainnet
+   * @param {number} [nonce=null] specified transaction index. Will fetch correct value if not specified.
+   */
+  const checkTransactionExistenceOnSafeServer = async function (masterSafe, transaction, network, nonce = null) {
+    const safeTxGas = await estimateGas(masterSafe, transaction)
+    const transactionHash = await masterSafe.getTransactionHash(
+      transaction.to,
+      transaction.value,
+      transaction.data,
+      transaction.operation,
+      safeTxGas,
+      baseGas,
+      0,
+      ZERO_ADDRESS,
+      ZERO_ADDRESS,
+      nonce
+    )
+    console.log(`The reproduced transaction has the gnosis-safe hash ${transactionHash} and nonce ${nonce}`)
+    const endpoint = `https://safe-transaction.${network}.gnosis.io/api/v1/transactions/${transactionHash}/`
+    await axios.get(endpoint).catch(function (error) {
+      if (error.response.data.detail === "Not found.") {
+        throw new Error("The reproduced transaction was not found. There is something wrong. Contact the proposer.")
+      }
+      throw new Error("Error while talking to the gnosis-interface: " + JSON.stringify(error.response.data))
+    })
+    const interfaceLink = `https://${linkPrefix[network]}gnosis-safe.io/app/#/safes/${masterSafe.address}/transactions`
+    console.log(
+      `The reproduced transaction was is already proposed! You can sign the transaction with nonce ${nonce} here: ${interfaceLink}`
```

Comment:
```suggestion
      `The transaction built from the verification parameters matches a transaction in the web interface. You can sign the transaction with nonce ${nonce} here: ${interfaceLink}`
```

Reply:
I really like the new suggested wording here!